pkg/scripts/os_centos.go

```diff
@@ -52,6 +52,13 @@ gpgcheck=1
 repo_gpgcheck=0
 gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
 EOF
+
+os_name=$(cat /etc/os-release | head -n1 | cut -d \" -f2)
+os_version=$(cat /etc/os-release | head -n2 | tail -1 | cut -d \" -f2)
```

Comment:
it's more robust to "source" that file since line number can be different you never know for sure what would you get.